lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1380,6 +1380,11 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
```

Comment:
Looks like we are including the test telecom update in this pr. https://github.com/cerner/fhir.cerner.com/pull/561

Reply:
Changed the base branch for remove operation from main to test operation branch https://github.com/cerner/fhir.cerner.com/pull/561
Now Remove operation PR contains only remove changes.

Reply:
talked in person. merging test and remove separately to main since the PR's were created originally in that way.